### ISSUE TITLE
Questions about batch size settings when fine-tuning CosmoPredict2OFT

### ISSUE THREAD

**Issue by abyss0271** [starVLA/starVLA#367]
Hello StarVLA team,

First of all, thank you for your amazing work and for open-sourcing this project!
I am currently trying to fine-tune **CosmoPredict2OFT** using the official code provided in the repository. Due to limited computational resources, I experimented with three different hardware and hyperparameter settings:
Setting 1: 4x 24GB GPUs (RTX 4090), per_batch_size=1, gradient_accumulation_steps=16
Setting 2: 4x 24GB GPUs (RTX 4090), per_batch_size=1, gradient_accumulation_steps=1
Setting 3: 2x 48GB GPUs, per_batch_size=4, gradient_accumulation_steps=16
All other configurations were kept exactly identical to those specified in WM4A.md.

The Issue:
After training, I evaluated the models on the 4 LIBERO tasks. **Unfortunately, the success rate is 0% across all tasks for all three of my training settings.**
I have verified that my evaluation code and environment are correct, as I successfully evaluated the official Qwen3-VL-OFT-LIBERO-4in1 checkpoint and achieved the success rates reported in your documentation.
**Below are the wandb curves for Setting 2 (Blue curve, which had an interruption and was resumed from a checkpoint) and Setting 3 (Red curve).**

<img width="2035" height="352" alt="Image" src="https://github.com/user-attachments/assets/d7977fc5-b2b3-4fc3-8170-cbde40a1661e" />

Since the official documentation recommends per_batch_size=16, I suspect the issue might be related to my batch size and gradient accumulation setups. I would sincerely appreciate your insights on the following questions:
Hardware & Batch Size Recommendations: **When using per_batch_size=16, how many GPUs does the official team recommend or typically use?** Have you tested the training performance/stability across different GPU counts and effective batch sizes? If so, would it be possible to share those insights?
Reference Checkpoints: **Would it be possible to provide a fine-tuned checkpoint for CosmoPredict2OFT along with its evaluation metrics on LIBERO?** Having a reference checkpoint would be incredibly helpful for debugging and aligning our results.

Thank you very much for your time and assistance. I look forward to your reply!
